client/src/components/CalendarPage.js

```diff
@@ -133,6 +133,29 @@ class CalendarPage extends React.Component {
     this.setState({ modal: false })
   }
 
+  cancelBooking = (student_id,lecture_id)=> {
+    API.cancelBooking(student_id,lecture_id)
+    .then((res)=>{
+      // GIVE FEEDBACK TO USER + change status of selected lecture
+      console.log(res);
+      // TODO this could be a function to remove duplication
+      let startOfWeek = moment().day(1).format("YYYY-MM-DD");
```

Comment:
This should be linked to the page of the calendar that we are watching, not the date of today, right?

Reply:
Yes, in the [calendar-week-by-week](https://github.com/se2team3/PULSeBS/blob/calendar-week-by-week/client/src/components/CalendarPage.js#L26) branch this is updated and the function that directly calls the API operates on updated dates. Here something similar should be done

Reply:
ok, is there a pull request for that? It seems that it has been merged in #30 hide-fields-and-add-tests